manifests/d/DirkJansen/MailCheck2/2.140 (Build 579)/DirkJansen.MailCheck2.installer.yaml

```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.6.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: DirkJansen.MailCheck2
+PackageVersion: 2.140 (Build 579)
+InstallerType: inno
+Installers:
+- Architecture: x64

```

Comment:
AppsAndFeaturesEntries are needed for proper matching on DisplayVersion -

#### x64
```
--> Comparing ARP Entries

DisplayName                                 DisplayVersion          Publisher   ProductCode   Scope
-----------                                 --------------          ---------   -----------   -----
MailCheck 2 version 2.140 (Build 579-64bit) 2.140 (Build 579-64bit) Dirk Jansen MailCheck_is1 Machine
```

#### x86
```
--> Comparing ARP Entries

DisplayName                           DisplayVersion    Publisher   ProductCode   Scope
-----------                           --------------    ---------   -----------   -----
MailCheck 2 version 2.140 (Build 579) 2.140 (Build 579) Dirk Jansen MailCheck_is1 Machine
```

Reply:
Thanks for the hint! I added DisplayVersion entries for both installer urls.